metadata.rb

```diff
@@ -5,7 +5,7 @@
 description      'Installs/Configures mailhog'
 version          '0.1.0'
 
-depends          'runit', '>= 0.0.0'
+depends          'runit', '= 1.7.4'
```

Comment:
The standard behavior of runit < 1.7.6 doesn't take care of automatically restarting the service, if its' configuration has changed. Due to this reason, the service can be restarted each time after enabling it directly in the recipe.

This is not possible in Runit >= 1.7.6, because it runs in the loop trying to restart the service.
However just enabling the service with Runit >= 1.7.6 doesn't accept the configuration changes.